cli_connection.py

```diff
@@ -0,0 +1,92 @@
+import asyncio
+import logging
+import sys
+
+from aapns.connection import Connection, Request, create_ssl_context
+
+USAGE = (
+    "Usage: python cli_connection.py [opts] device_token cmd message-1 message-2 ..."
+)
+
+
+async def send_several(ssl_context, base_url, requests):
+    async with Connection(base_url, ssl=ssl_context) as c:
+        tasks = []
+        for r in requests:
+            logging.info("Sleeping a bit")
+            await asyncio.sleep(1)
+
+            async def post(r):
+                try:
+                    logging.info("%s", r)
+                    logging.info("%s", await c.post(r))
+                except Exception as rv:
+                    logging.info("%s", rv)
+
+            # FIXME 2. Optional header fields
+            # Apns-Id, Apns-Expiration, Apns-Topic, Apns-Collapse-Id
+            tasks.append(asyncio.create_task(post(r)))
+        await asyncio.gather(*tasks)
+
+
+if __name__ == "__main__":
```

Comment:
How about using `click` or `argparse` to handle arguments?

Reply:
I think i'll remove these tools before long... I'll keep your comment open until then.

Reply:
I think it should be okay to leave this tool so lib developer can test things

Reply:
Done